doc/users/cmake_dealii.html

```diff
@@ -894,6 +895,52 @@ <h3> Selecting a compiler </h3>
     </p>
 
 
+    <a name="compileccache"></a>
+    <h3> Compiling using Ccache </h3>
+    <p>
+      <a href="https://ccache.dev/">Ccache</a> is a compiler caching tool. It 
+      accelerates recompilation by caching previously executed compilations 
+      and subsequently detecting when the same compilation is being performed
+      again.
+    </p>
+    
+    <p>
+      It is possible for the library to be built using the Ccache tool.
+      This may be useful for those who frequently rebuild the library, as it
+      caches the output of the compilation (the C++ components, at least).
+      The next time that the library is built, if the compilation objects remain
+      the same then rebuilding them can be avoided as the results will be taken
+      from directly the cache. Anecdotally, Ccache is not well suited to
+      everybody's workflow, so it may take some experimentation to determine if
+      it's effective and helpful, or offers no particular gain to you.
+    </p>
+
+    <p>
+      Building with Ccache can be enabled directly via the command line. 
+      When using a version of CMake that is greater or equal to 3.4, then using
+      Ccache is a little easier. The above may be compressed into a single
+      configure-time argument
+<pre class="cmake">
+  cmake -DCXX_COMPILER_LAUNCHER="ccache" &lt;...&gt;
```

Comment:
Users sometimes get confused when we do things like `./path/to/library` (they enter it literally). Could we just have the flag alone here to make things slightly less ambiguous?

Reply:
Ok, I've done now this. But note that the way that I wrote it was more or less consistent with the rest of the documentation on this page.